qct_parse/qct_parse.py

```diff
@@ -775,6 +778,13 @@ def main():
 		config = configparser.RawConfigParser(allow_no_value=True)
 		dn, fn = os.path.split(os.path.abspath(__file__)) # grip the dir where ~this script~ is located, also where config.txt should be located
 		# assign config based on bit depth of tag values 
+		config_file_path = os.environ.get(CONFIG_ENVIRONMENT_VARIABLE_NAME)
+		if not config_file_path:
+			raise AttributeError(
+				f'{CONFIG_ENVIRONMENT_VARIABLE_NAME} is not set. '
+				f'This is needed to locate config files.'
+			)
+
 		if bit_depth_10:
 			config.read(os.path.join(dn,"qct-parse_10bit_config.txt")) # read in the config file
```

Comment:
need to change the config.read() call as well, "config.read(os.path.join(config_file_path,"qct-parse_10bit_config.txt"))"

Reply:
I've moved it so that the config files are now included in the package and setting QCT_PARSE_CONFIG_DIRECTORY changes the path the config files are found. This would be useful if you want to change the defaults of these config files.